duckduckgo/duckduckgo-privacy-extension
# Remove verbose background image SCSS

**Reviewer:** @MariagraziaAlastra 
**CC:** @jdorweiler 

## Description:

This removes the old way of defining background images across platforms, replacing it with the `cross_platform_bg_image` mixin.

That way there's less typing, less misdirection, and less room for dev error. The compiled CSS should be pretty much the same.

This supports https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/195. (I've been meaning to do it for a while.)

## Steps to test this PR:

1. Use the extension normally, visit a bunch of the subviews.
2. Pay particular attention to the hero icon on the main popup, the tracker networks on the subview and the tracker top offenders.
3. Images should look exactly the same as before

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
